server/model/entities/Sniffer.ts

```diff
@@ -58,6 +58,12 @@ export class Sniffer {
   @Column("uuid", { name: "user_id", nullable: true })
   userId: string;
 
+  @Column("boolean", {
+    name: "is_mocking_enabled",
+    default: false,
```

Comment:
Default should be true please

```suggestion
    default: true,
```